desktop-widgets/tab-widgets/TabDiveInformation.h

```diff
@@ -24,12 +24,16 @@ private slots:
 	void on_visibility_valueChanged(int value);
 	void on_airtemp_editingFinished();
 	void on_watertemp_editingFinished();
+	void on_waterTypeCombo_activated(int index);
 private:
 	Ui::TabDiveInformation *ui;
+	bool manualDive;
```

Comment:
I'm not convinced that we need this member variable: It is only queried in one place. Can't we simply do the check there?

Reply:
I get a segmentation fault if I access dive->dc.model directly in the code. In general I have had quite a bit of problems of crashes when accessing current_dive's components.